Sources/Sentry/include/SentryBinaryImageCache.h

```diff
@@ -0,0 +1,25 @@
+#import "SentryDefines.h"
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SentryBinaryImageInfo : NSObject
+@property (nonatomic, strong) NSString *name;
+@property (nonatomic) uint64_t address;
+@property (nonatomic) uint64_t size;
+@end
+
+@interface SentryBinaryImageCache : NSObject
```

Comment:
`m`: I think a comment here explaining why we need to sync the binary image cache from the c layer to Objective-C would make sense.